src/frontend/screens/Login/index.css

```diff
@@ -16,61 +18,186 @@
 }
 
 .loginContentWrapper {
-  width: 100%;
+  width: 75%;
   height: 100%;
   display: flex;
-  align-items: center;
   justify-content: center;
   flex-direction: column;
   grid-column: 1;
   grid-row: 1;
   z-index: 1;
+  margin: 0 auto;
 }
 
 .runnerList {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   align-items: center;
   justify-content: center;
   position: relative;
   user-select: none;
-  width: 100%;
   flex-wrap: wrap;
+  background-color: var(--background);
+  padding: var(--wrapper-padding);
+  border-radius: 10px;
+  margin-top: 20px;
+  min-width: 300px;
+}
+
+.languageSelector {
+  position: absolute;
+  top: -75px;
```

Comment:
maybe use `bottom: calc(100% + 10px)` here? with -75px is easy for changes to create issues, this `help translate Heroic` was fixed (it was missing) and it overlaps now for example:
![image](https://user-images.githubusercontent.com/188464/197353158-b9afb6ab-baf8-44cf-9b48-ff53e60fa635.png)

Reply:
Here's what that looks like with those fixes implemented: 
<img width="621" alt="Screenshot 2022-10-22 at 5 18 07 PM" src="https://user-images.githubusercontent.com/99497246/197362805-30f603b6-18f3-44ea-b812-6d9dbf7d6af0.png">